admin_manual/installation/system_requirements.rst

```diff
@@ -5,42 +5,92 @@ System Requirements
+For *best performance*, *stability*, *support*, and *full functionality* we
+officially recommend and support:
 
-* Ubuntu 16.04
-* MySQL/MariaDB
-* PHP 5.6+ and 7.0+
-* Apache 2.4 with mod_php
+================= =============================================================
+Platform          Options
+================= =============================================================
+Operating System  Ubuntu 16.04
+Database          MySQL or MariaDB 5.5+
+Web server        Apache 2.4 with mod_php
+PHP Runtime       PHP (5.6+ or 7.0+)
+================= =============================================================
 
 Supported Platforms
 -------------------
 
-* Server: Linux (Debian 7 and 8, SUSE Linux Enterprise Server 12 and 12 SP1, Red Hat Enterprise Linux/Centos 6.5 and 7 (7 is 64-bit only), Ubuntu 14.04 LTS, 16.04 LTS)
-* Web server: Apache 2.4 with mod_php
-* Databases: MySQL/MariaDB 5.5+; Oracle 11g (ownCloud Enterprise edition only); PostgreSQL
-* PHP 5.6+ required
-* Hypervisors: Hyper-V, VMware ESX, Xen, KVM
-* Desktop: Windows 7+, Mac OS X 10.7+ (64-bit 
-  only), Linux (CentOS 6.5, 7 (7 is 64-bit only), Ubuntu 12.04+, Fedora 20+, openSUSE 12.3+, Debian 7 & 8).
-* Mobile apps: iOS 7+, Android 4+
-* Web browser: IE11+ (except Compatibility Mode), Firefox 14+, Chrome 18+, 
-  Safari 5+
-
-See :doc:`source_installation` for minimum software versions for installing 
+If you are not able to use one or more of the above tools, the following
+options are also supported. 
+
+Server
+^^^^^^
+
+- Debian 7 and 8
+- SUSE Linux Enterprise Server 12 and 12 SP1
+- Red Hat Enterprise Linux/Centos 6.5 and 7 (**7 is 64-bit only**)
+- Ubuntu 14.04 LTS
+
+Web Server
+^^^^^^^^^^
+
+- Apache 2.4 with mod_php
+
+Databases
+^^^^^^^^^
+
+- Oracle 11g (**Enterprise edition only**)
+- PostgreSQL
+
+Hypervisors 
+^^^^^^^^^^^
+
+- Hyper-V
+- VMware ESX
+- Xen
+- KVM
+
+Desktop
+^^^^^^^
+
+- Windows 7+
+- Mac OS X 10.7+ (64-bit only)
+- Linux (CentOS 6.5, 7 (7 is 64-bit only)
+- Ubuntu 12.04+
+- Fedora 20+
+- openSUSE 12.3+
+- Debian 7 & 8
+
+Mobile 
+^^^^^^
+
+- iOS 7+
+- Android 4+
+
+Web Browser 
+^^^^^^^^^^^
+
+- IE11+ (except Compatibility Mode)
+- Firefox 14+
+- Chrome 18+
+- Safari 5+
+
+See :doc:`source_installation` for minimum software versions for installing
 ownCloud.
 
 Database Requirements for MySQL / MariaDB
 -----------------------------------------
 
-The following is currently required if you're running ownCloud together with a MySQL / MariaDB database:
+The following are currently required if you're running ownCloud together with a MySQL or MariaDB database:
 
 * Disabled or BINLOG_FORMAT = MIXED configured Binary Logging (See: :ref:`db-binlog-label`)
-* InnoDB storage engine (MyISAM is not supported, see: :ref:`db-storage-engine-label`)
-* "READ COMMITED" transaction isolation level (See: :ref:`db-transaction-label`)
\ No newline at end of file
+* InnoDB storage engine (The MyISAM is not supported, see: :ref:`db-storage-engine-label`)
```

Comment:
If you write "The MyISAM is not supported..." then it needs to be said fully:
"The MyISAM storage engine is not supported..."
otherwise just "MyISAM is not supported..." seems fine.